ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/ip_access_lists.py

```diff
@@ -28,7 +28,46 @@ def ip_access_lists(self) -> list | None:
                 for acl in interface_acls.values():
                     append_if_not_duplicate(ip_access_lists, "name", acl, context="IPv4 Access lists for SVI", context_keys=["name"])
 
-        if self._filtered_internet_exit_policies:
+        zscaler_policy = False
+        interface_ips = set()
+        for ie_policy in self._filtered_internet_exit_policies:
+            if ie_policy["type"] == "zscaler":
+                zscaler_policy = True
+            elif ie_policy["type"] == "direct":
+                for connection in ie_policy["connections"]:
+                    interface_ips.add(connection["source_interface_ip_address"])
+
+        if interface_ips:
+            interface_ips = sorted(interface_ips)
+            entries = []
+            for i, interface_ip in enumerate(interface_ips):
+                entries.append(
+                    {
+                        "sequence": 10 + i * 10,
+                        "action": "deny",
+                        "protocol": "ip",
+                        "source": interface_ip,
```

Comment:
We should only match the interface IP itself and not the whole subnet. This also removes an issue where we printed 10.10.10.1/31 instead of 10.10.10.0/31.

```suggestion
                        "source": f"host {interface_ip.split('/', maxsplit=1)[0]}",
```